### DISCUSSION TITLE
Is Coverity Scan still valid?

### DISCUSSION THREAD

**Discussion by lucydodo** [sqlitebrowser/sqlitebrowser#3437] [General]
https://scan.coverity.com/projects/sqlitebrowser-sqlitebrowser
Shows that the last analysis was on Dec 27, 2021.

Now that we have [CodeQL](https://github.com/sqlitebrowser/sqlitebrowser/pull/3388) in our project, shouldn't it be okay to remove the docs, workflow, etc for Coverity Scan?

**Comment by justinclift** [comment]
Good question.  I *think* there might be newer Coverity scans happening.

@MKleusberg You were looking at this (ages ago).  What do you reckon? :smile:
